nx.json

```diff
@@ -6,7 +6,7 @@
   },
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32338&commitId=9c93263d6916d1b2424ab11bb097d9be445b8d36&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>